pkg/scalers/prometheus_scaler.go

```diff
@@ -134,6 +144,16 @@ func parsePrometheusMetadata(config *ScalerConfig) (meta *prometheusMetadata, er
 		meta.cortexOrgID = val
 	}
 
+	meta.ignoreNullValues = defaultIgnoreNullValues
+	if val, ok := config.TriggerMetadata[ignoreNullValues]; ok && val != "" {
+		ignoreNullValues, err := strconv.ParseBool(val)
+		if err != nil {
+			return nil, fmt.Errorf("err incorrect value for ignoreNullValues given: %s, " +
+				"please use ture or false", val)
```

Comment:
```suggestion
				"please use true or false", val)
```
@bamboo12366 there is still typo

Reply:
sorry...